apps/src/studio/studio.js

```diff
@@ -1511,7 +1511,9 @@ function updateItems() {
     }
   }
   Studio.sprite.forEach(sprite => {
-    sprite.update();
+    if (sprite.visible) {
+      sprite.update();
```

Comment:
I just added this call in my previous sprite group PR, and all it does is call Item's `update()` method to drive group behavior (and draw a debug rect). I suppose it makes more sense to move the check into the unnecessary parts of `update()`, in case anybody else adds code there expecting it to always run.

Updated the code accordingly.